test/end-to-end/README.md

```diff
@@ -105,18 +105,18 @@ describe('View Recipe Page', function () {                // Which page does thi
     after((done) => {                                     // Scenario de-provision, like apply a snapshot
     });
 
-    context('Menu Nav Bar Check', () => {                 // Element or elements to be test
-      it('should show a recipe name', (done) => {         // Expected result description
+    context('Menu Nav Bar Check #Acceptance', () => {     // Test content and test type
+      it('should show a recipe name @Welder-Web-40003', (done) => {    // Case description and case ID
```

Comment:
Test type is fine, but IMO we don't need the case ID because the titles are descriptive enough. I don't see how ID add any new information to the mix.